server/src/main/java/de/pnp/manager/component/item/Item.java

```diff
@@ -86,9 +87,19 @@ public class Item extends DatabaseObject implements IItem, IUniquelyNamedDataObj
      */
     protected final String note;
 
-    public Item(ObjectId id, String name, ItemType type, ItemType subtype, String requirement,
-        String effect,
-        ERarity rarity, int vendorPrice, int tier, String description, String note) {
+    /**
+     * The maximal amount of this item that can be in one {@link ItemStack}.
+     */
+    protected final int maximalStackSize;
+
+    /**
+     * The minimal amount of this item that has to be in one {@link ItemStack}.
+     */
+    protected final int minimalStackSize;
```

Comment:
Does a minimalStackSize of 0 even make sense? (i.e. should there be an assertion that minimalStackSize > 0?